core/src/main/scripts/importer/validateData.py

```diff
@@ -2646,6 +2653,14 @@ def process_metadata_files(directory, portal_instance, logger, relaxed_mode):
                 logger.warning(
                 'Citation is required when giving a pubmed id (pmid).')
 
+        # TBD: if we are processing a gene panel meta file,
+        # we should validate the gene list.  Currently checkGeneIdentification
+        # validation is buried in Validator class which makes this difficult.
+        # Another issue to consider is whether or not to raise a flag when a
+        # gene panel getting validated possesses the same stable id as an existing
+        # gene panel within the database.
+        #if meta_file_type == cbioportal_common.MetaFileTypes.GENE_PANEL:
+                
```

Comment:
I agree with the proper fix.  The Java code is going away in the future as we will be replacing it with the spring-batch implementation of the scripts package, so I don't want to prioritize this.  I will go ahead and remove the comment.